externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/SparkSQLEngine.scala

```diff
@@ -76,6 +75,8 @@ object SparkSQLEngine extends Logging {
 
   private val countDownLatch = new CountDownLatch(1)
 
+  val replOutputDir = Utils.createTempDir(namePrefix = "repl").toFile
```

Comment:
Why not directly `val replOutputDir = Utils.createTempDir(namePrefix = "repl").toFile. getAbsolutePath`?

Reply:
thanks